### ISSUE TITLE
[CoE Starter Kit - BUG] Cannot save-as Governance Component flows

### ISSUE THREAD

**Issue by dchrist27** [microsoft/coe-starter-kit#4760]
### Describe the issue

In the past (Nov 22 CoE update) I was able to save-as Governance Component flows to MyFlows then bring those over into my own solution (instead of creating a solution layer on these flows).

### What solution are you experiencing the issue with?

Governance

### What solution version are you using?

3.6

### What app or flow are you having the issue with?

All inactivity flows

### Anything else?

When I attempt to save-as I get a prompt that its a child flow.

![image](https://user-images.githubusercontent.com/52799471/217661259-e7210bd3-bf1c-400f-bf04-b8d6ee0ee750.png)

This wasn't the case back in November.

**Comment by manuelap-msft** [comment]
Hello,

this is a known limitation of the Power Automate "Save As" capability which is not supported for flows that call child flows.

Manuela

**Comment by dchrist27** [comment]
Maybe this might help.  So back in Sept and Nov I was able to save-as or maybe rename these governance flows.   Note...they are not managed and reside in a new solution I created.

![image](https://user-images.githubusercontent.com/52799471/217838201-2f802c66-c3ce-4352-a229-910f77df25b8.png)

If I try add existing | automation | cloud flow it just seems to copy the flow (its also still flagged as managed)

![image](https://user-images.githubusercontent.com/52799471/217838501-72254029-64b1-4560-9f9c-4a3288f6bf56.png)
